frontend/src/components/Card/card-styles.js

```diff
@@ -4,4 +4,6 @@ export const card = (theme) => css`
   background-color: ${theme.color.neutral[0]};
   border-radius: 16px;
   box-shadow: 0px 12px 24px -12px rgba(0, 25, 68, 0.2);
+  height: max-content;
+  padding: 14px;
```

Comment:
move this line to task-card-styles.js, since adding it on card-styles.js would mean that all of the cards will have 14px padding, which is not the case